prow/images/cla-assistant-external-plugin/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# This dockerfile is meant to use on externally built binary using command:
+# GOOS=linux GOARCH=amd64 go build -o ./plugin .
+# then use docker build -t cla-assistant .
+# NOT MEANT FOR AUTOBUILDS
+
+FROM alpine:3.15.0
+
+ARG commit
+ENV IMAGE_COMMIT=$commit
+LABEL io.kyma-project.test-infra.commit=$commit
+COPY ../../../development/external-plugins/cla-assistant/plugin /
```

Comment:
That's true this file was meant to be in the same place with code. It will not work from here and is not meant to work.

I would like to experiment with some other tooling that would allow us to not use any Dockerfiles for go programs. If that's not the problem I'd keep this (and needs-tws) Dockerfile along with the code.

Reply:
Please explain this in a file so if anyone will work with this file will know why it's here and will not have same concerns as Ammar.

Reply:
Then maybe let's keep this Dockerfile along with the code in the external plugins directory

Reply:
You wrote you need this COPY to experiment with some tool which doesn't need a Dockerfile. Why changing a location of Dockerfile is needed then? Till now we keep Dockerfiels in prow/images and it was working fine. Maybe the best would be to make this as usual, to not confuse anyone. Experiment in separate PR. If you think changing location of Dockerfiles is needed bring this in separate issue so we can discuss and plan migration for whole test-infra.

Reply:
The explanation and purpose of this file was already described in the Dockerfile. I've further detailed on how to work with it and changed the COPY statement to be valid, as per review comment.